rq/worker.py

```diff
@@ -950,6 +960,18 @@ def handle_exception(self, job, *exc_info):
             if not fallthrough:
                 break
 
+    def success_job_callback(self,job,*exc_info):
```

Comment:
If the job is successfully executed, there's no need to pass `exc_info` as arguments.